### DISCUSSION TITLE
5%-95% Confidence interval

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1380] [Questions & Problems]
Hello everyone, I am using PK-sim for model building recently, and I want to use the 90% confidence interval of the group. I would like to consult whether the software can output directly or calculate through other methods. Looking forward to your answers!
![5%-95%-1](https://user-images.githubusercontent.com/95362728/201555343-0a9942f3-fe27-4d91-9ffd-7046eebbb0cc.jpg)
![5%-95%-2](https://user-images.githubusercontent.com/95362728/201555351-4d8b6bac-dd84-4242-827e-14263b0c9f61.jpg)

**Comment by PavelBal** [comment]
This will be available in version 11.1, soon to be released ;)

**Reply by WTY-tt** [reply]
Thank you very much for your answer, and I would like to know when this function will be released？

**Comment by PavelBal** [comment]
A release of the beta verison is planned for this or next week.

**Reply by WTY-tt** [reply]
Thank you. I am hoping for this new verison.

**Reply by msevestre** [reply]
I am hoping for this new version too
